src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -112,8 +114,12 @@ export class Conversations extends React.Component<Props, State> {
         initialListSize={10}
         scrollEventThrottle={10}
         onEndReachedThreshold={10}
+        enableEmptySections={true}
         refreshControl={
-          <RefreshControl refreshing={this.state.fetchingNextPage} onRefresh={() => this.fetchData(false)} />
+          <RefreshControl
+            refreshing={this.state.fetchingNextPage && this.state.initialLoadDone}
+            onRefresh={() => this.fetchData(false)}
```

Comment:
If this is all that's needed to refresh, that's awesome, but on Home we had to do a [`forceFetch` of each individual module](https://github.com/artsy/emission/blob/master/src/lib/Containers/Home.tsx#L62) to get accurate results so that's something to look out for.